client-core/src/transaction_builder/raw_transfer_transaction_builder.rs

```diff
@@ -356,14 +356,25 @@ where
         let input_value = self.total_input_amount()?;
         let output_value = self.total_output_amount()?;
 
-        // TODO: Include fee in calculation?
         if input_value < output_value {
             return Err(Error::new(
                 ErrorKind::VerifyError,
                 "Output amount exceed input amount",
             ));
         }
 
+        // check fee
+        let fee_in_tx = (input_value - output_value)
+            .chain(|| (ErrorKind::VerifyError, "Invalid amount of output or input"))?;
```

Comment:
how about
insufficient amount ,
because intput, output can be difficult for users

Reply:
I'll fix it.